src/pages/NewListing.jsx

```diff
@@ -339,6 +345,14 @@ const NewListing = () => {
   const haveEnoughZoom =
     wallet?.zoomBalance > numberOfSelectedCards * data?.zoomBurnFee
 
+  const { data: currentAllowance, isLoadingAllowance } = useGetZoomAllowanceQuery(
+    wallet.address,
+    contracts.ZoomContract
+  )
+
+  const exceedZoomAllowance = numberOfSelectedCards * data?.zoomBurnFee > currentAllowance
+  console.log('exceedZoomAllowance', exceedZoomAllowance);
```

Comment:
nit, can we remove this console log? It will repeatedly log during re-render

Reply:
Yes. Of coures. I will remove it.